packages/core/src/useForm/useFormActions.ts

```diff
@@ -107,6 +108,7 @@ export function useFormActions<TForm extends FormObject = FormObject, TOutput ex
 
       const result = await onSuccess(withConsumers(output), { event: e, form: e?.target as HTMLFormElement });
       isSubmitting.value = false;
+      submitCount.value += 1;
```

Comment:
I think it should increment regardless if the submit was successful or not, maybe we should name it `submitAttempts` to make it clearer? What do you think?